spd/configs.py

```diff
@@ -161,6 +161,14 @@ def all_module_patterns(self):
         description="Coefficient for recon loss with stochastically sampled masks on one layer at "
         "a time",
     )
+    ci_recon_subset_coeff: NonNegativeFloat | None = Field(
```

Comment:
I'd change this to ci_masked_recon_subset_coeff now since we're going to have to do it at some stage.